src/signals/incident/definitions/wizard-step-2-vulaan/wonen.ts

```diff
@@ -778,6 +778,200 @@ const vakantieverhuur = {
+        intimidatie: 'Intimidatie',
+        huurcontract: 'Huurcontract',
+        bemiddelingskosten: 'Bemiddelingskosten',
+        arbeidsmigrant: 'Arbeidsmigrant',
+        servicekosten: 'Servicekosten',
+        overige: 'Overige (iets anders namelijk)',
+      },
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.RadioInput,
+  },
+
+  // Discriminatie
+  extra_wonen_verhuurderschap_discriminatie: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'discriminatie',
+      },
+      label:
+        'Bent u gediscrimineerd tijdens het verhuurproces? We bedoelen in dit geval of u een woning niet heeft gekregen omdat u tot een specifieke groep behoort. Op welk grond bent u gediscrimineerd? Kies uit:',
+      shortLabel: 'Discriminatie groep',
+      pathMerge: 'extra_properties',
+      values: {
+        ras: 'Ras',
+        geloofsovertuiging: 'Geloofsovertuiging',
+        politieke_overtuiging: 'Politieke overtuiging',
+        gesalcht: 'Geslacht',
+        nationaliteit: 'Nationaliteit',
+        seksuele_gerichtheid: 'Seksuele gerichtheid',
+        burgelijke_staat: 'Burgerlijke staat',
+        handicap: 'Handicap',
+        chonische_ziekte: 'Chronische ziekte',
+        anders: 'Een andere reden, namelijk ',
+      },
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.CheckboxInput,
+  },
+  extra_wonen_verhuurderschap_discriminatie_anders: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_discriminatie: 'anders',
+      },
+      label: 'Een andere reden, namelijk:',
+      shortLabel: 'Andere reden',
+      pathMerge: 'extra_properties',
+      type: 'url',
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.TextInput,
+  },
+  extra_wonen_verhuurderschap_discriminatie_bewijs: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'discriminatie',
+      },
+      label: 'Aanvullende informatie:',
+      shortLabel: 'Bewijs discriminatie',
+      pathMerge: 'extra_properties',
+      values: {
+        bewijs:
+          'Kunt u de discriminatie tijdens het verhuurproces aantonen door bijvoorbeeld een afwijzing van de woning of de selectiecriteria voor het verkrijgen van de woning? (Zo ja, voeg deze stukken toe aan de bijlage)',
+      },
+    },
+    render: QuestionFieldType.CheckboxInput,
+  },
+
+  // Intimidatie
+  extra_wonen_verhuurderschap_intimidatie: {
+    meta: {
+      ifOneOf: {
+        extra_wonen_verhuurderschap_onderwerp: 'intimidatie',
+      },
+      label:
+        'Bent u bijvoorbeeld tijdens het verhuurproces door lichamelijk of psychisch geweld of bedreiging met geweld gedwongen om iets te doen of juist iets niet te doen?',
+      shortLabel: 'Intimidatie',
+      pathMerge: 'extra_properties',
+    },
+    options: {
+      validators: ['required'],
+    },
+    render: QuestionFieldType.TextInput,
```

Comment:
Ik zou hier een textareainput nemen (die kan alleen niet verplicht worden geloof ik). 1 regel om te zeggen dat ik fysiek bedreigt ben is wel heel weinig.